modal/_container_entrypoint.py

```diff
@@ -557,11 +590,13 @@ def main(container_args: api_pb2.ContainerArguments, client: Client):
         # Initialize the function
         with function_io_manager.handle_user_exception():
             imp_fun = import_function(container_args.function_def, ser_cls, ser_fun)
+            if imp_fun.stub:
+                _container_app = synchronizer._translate_in(container_app)
+                _client = synchronizer._translate_in(client)
```

Comment:
i don't think you need to translate in here? the `_hydrate_function_handles` is a synchronized function that will do the translation automatically

Reply:
`imp_fun.stub` is an unwrapped stub (to avoid having different interfaces when the user has an AioStub vs Stub). In that case it won't translate the inputs right?